resources/packs/gcp/gcp.lr

```diff
@@ -1040,4 +1042,112 @@ private gcp.project.cluster.nodepool.config.confidentialNodes @defaults("enabled
   id string
   // Whether to use confidential nodes
   enabled bool
-}
\ No newline at end of file
+}
+
+// GCP KMS resources
+private gcp.project.kms {
+  // Project ID
+  projectId string
+  // The available locations for the service
+  locations() []string
+  // The list of keyrings in the current project
+  keyrings() []gcp.project.kms.keyring
+}
+
+// GCP KMS keyring
+private gcp.project.kms.keyring @defaults("name") {
+  // Internal ID
+  id string
+  // Project ID
+  projectId string
+  // The name of the keyring
+  name string
+  // Creation timestamp
```

Comment:
```suggestion
  // Time created
```

Reply:
In other places we have called this `Creation timestamp`. Not only for GCP resources but also in other resources. We should decide if we want all of them to be changed or not. I will leave this one as is for now